app/main.py

```diff
@@ -1,3 +1,59 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int or float) -> None:
```

Comment:
`int or float`
Is not correct use or Union[int, float] or int | float
Fix in all places